### ISSUE TITLE
Add utility to require/register all models

### ISSUE THREAD

**Issue by johnhof** [johnhof/mongoman#13]
a crude example https://github.com/johnhof/IceBuddy/blob/master/api/lib/helpers.js#L29

a few requirements in addition to whats listed above
- allow option to require all JS files in a flat directory
- allow option to require all JS files in a tree which match a regex
- [consider] allow option to provide a raw JS object as an export. 

eg:

``` javascript
var Mon = require('mongoman');
module.exports = {
  schema: {
    name: Mon.string().required.fin(),
    password: Mon.string().required().password.fin()
  }
  options: {
    virtuals: {
      // ...
    }
  }
}
```

**Comment by johnhof** [comment]
https://github.com/johnhof/mongoman/pull/14

**Comment by DuaneGarber** [comment]
Resolved by #14
